tests/test_oas/test_view.py

```diff
@@ -278,7 +285,7 @@ async def test_pets_id_route_should_have_get_method(generated_oas):
                 "name": "day",
                 "required": False,
                 "schema": {
-                    "anyOf": [{"type": "integer"}, {"const": "now"}],
```

Comment:
Why did you change this line ? The CI doesn't pass. 

```
FAILED tests/test_oas/test_view.py::test_pets_id_route_should_have_get_method - AssertionError: assert {'parameters'...iption': ''}}} == {'parameters'...iption': ''}}}
...
                     'schema': {'anyOf': [{'type': 'integer'},
  -                                       {'const': 'now',
  +                                       {'const': 'now'}],
  ?                                                      ++
  -                                        'enum': ['now'],
  -                                        'type': 'string'}],
                                'default': 'now',
                                'title': 'day'}}],
```

Reply:
perhaps it's some mismatch of versions, because this failed my local pytests. I will revert this change as well as the datetime.

Reply:
I reverted this change